src/main/java/com/parsing/parsers/pdf/download/DownloaderPDF.java

```diff
@@ -25,8 +25,8 @@ public class DownloaderPDF {
     private static final String DIR_TO_SAVE_PDF = "/pdf/";
     private final RestTemplate restTemplate;
 
-    public Path downloadByUrl(URI uri, UUID uuid) {
-        String savePath = getPath() + uuid.toString() + ".pdf";
+    public Path downloadByUrl(URI uri, String id) {
```

Comment:
consider leaving it as UUID to provide a validation for a parameter

Reply:
Earlier we used for id type "uuid", now we are using "String".